### ISSUE TITLE
eth.signTransaction not working when Display Data = No

### ISSUE THREAD

**Issue by husticdf** [LedgerHQ/ledgerjs#214]
Ledger Nano S, Firmware version 1.4.2, MCU version 1.5
Ethereum App version 1.1.4
"ethereumjs-tx": "^1.3.7"
"@ledgerhq/hw-app-eth": "^4.21.0"
"@ledgerhq/hw-transport-node-hid": "^4.22.0"

Hi,

I am trying to sign Ethereum transactions using the Ledger Nano S and the LedgerHQ libraries.

When I configure the Ethereum application on the Ledger with "Display Data" = "Yes", the code that creates and signs the transaction works.

However, with "Display Data" = "No" the code fails with "TransportStatusError: Ledger device: Invalid data received (0x6a80)".

I am using the following code (works with Quokka):

```
import EthereumTx from "ethereumjs-tx"
import LedgerAppEth from "@ledgerhq/hw-app-eth"
import LedgerTransportNodeHid from "@ledgerhq/hw-transport-node-hid"

async function sign(txParams) {

  // get transaction hex
  const txHex = new EthereumTx(txParams).serialize().toString("hex")

  // connect with Ledger
  const app = new LedgerAppEth(await LedgerTransportNodeHid.create())

  // sign transaction using Ledger
  // EXCEPTION when "Display Data" = "No"
  const signature = await app.signTransaction("44'/60'/0'/0/0", txHex)

  // copy the signature parameters
  txParams.r = `0x${signature.r}`
  txParams.s = `0x${signature.s}`
  txParams.v = `0x${signature.v}`

  // create new transaction
  const signedTxHex = new EthereumTx(txParams).serialize().toString("hex")

  return signedTxHex
}

const data = "0x53f7ce2d000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000000a0fffffffffffffffffffffffffffffffffffffffffffffffffffffffffff3cad0000000000000000000000000000000000000000000000000000000000000001200000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"

const exParams = {
  data,
  gasLimit: 6e6,
  gasPrice: 3e9,
  to: "0xd22829946a6b2cc3f47b9e9b52e543dd425ed9c8",
  nonce: "0x14",
  chainId: 1,
  v: "0x01",
  r: "0x00",
  s: "0x00"
}

sign(exParams).then(console.log, console.error)
```

Thank you!
Matija

**Comment by btchip** [comment]
This should be fixed in ETH application version 1.1.6 - can you please recheck ?

**Comment by husticdf** [comment]
Works. Thank you very much!
